src/core_modules/capture-core/components/List/OnlineList/OnlineList.component.tsx

```diff
@@ -169,7 +161,7 @@ class Index extends React.Component<Props> {
             const cells = visibleColumns.map(column => (
                 <DataTableCell
                     key={column.id}
-                    align={Index.typesWithRightPlacement.includes(column.type) ? 'right' : 'left'}
+                    align={Index.typesWithRightPlacement.includes(column.type as any) ? 'right' : 'left'}
```

Comment:
Remove `as any`. When there is no TypeScript error to fix do not unnecessary change the code and keep it as close to the original code as possible